<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>План эксперимента</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Сигналы</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QListView" name="exp_list"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="button_new_signal">
                <property name="text">
                 <string>Новый</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_delete">
                <property name="text">
                 <string>Удалить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_add_exp">
                <property name="text">
                 <string>Добавить в план</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>План эксперимента</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListView" name="plan_list"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Название:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="exp_name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="button_load_exp">
                <property name="text">
                 <string>Загрузить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_edit_ticket">
                <property name="text">
                 <string>Править</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_up_plan">
                <property name="text">
                 <string>Вверх</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_down_plan">
                <property name="text">
                 <string>Вниз</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_delete_plan">
                <property name="text">
                 <string>Удалить</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_count_tasks">
       <property name="text">
        <string>Всего тикетов: 0   Всего запросов к плате: 0   Примерное время выполнения: 0 мин.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_view_requets">
         <property name="text">
          <string>Просмотреть запросы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_check_exp">
         <property name="text">
          <string>Проверить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_apply_exp">
         <property name="text">
          <string>Применить к ячейке</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_apply_all">
         <property name="text">
          <string>Применить ко всем</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_cancel_exp">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
